src/Components/CardLIst/index.jsx

```diff
@@ -0,0 +1,140 @@
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '4',
+        title: 'Title 4',
+        text: 'Latin words, combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable. The generated Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '5',
+        title: 'Title 5',
+        text: 'It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passage.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '6',
+        title: 'Title 6',
+        text: 'The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '7',
+        title: 'Title 7',
+        text: 'It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '8',
+        title: 'Title 8',
+        text: 'Cicero, written in 45 BC. book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '9',
+        title: 'Title 9',
+        text: 'Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for lorem ipsum will uncover many web sites still in their infancy.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '10',
+        title: 'Title 10',
+        text: 'by injected humour, or randomised words which look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+    ],
+    viewModeFlag: false,
+  });
```

Comment:
This state logically belongs to application, not card list.
Component should only contain what it should contain and do what it's supposed to do,

Reply:
Fixed